targets/PythonSdk/make.js

```diff
@@ -0,0 +1,407 @@
+            + tabbing + "PlayFabSettings.EntityToken = playFabResult[\"EntityToken\"][\"EntityToken\"] or PlayFabSettings.EntityToken\n"
+            + tabbing + "MultiStepClientLogin(playFabResult[\"SettingsForUser\"])\n";
+    else if (apiCall.result === "RegisterPlayFabUserResult")
+        return tabbing + "PlayFabSettings.ClientSessionTicket = playFabResult[\"SessionTicket\"] or PlayFabSettings.ClientSessionTicket\n"
+            + tabbing + "MultiStepClientLogin(playFabResult[\"SettingsForUser\"])\n";
+    else if (api.name === "Client" && apiCall.result === "AttributeInstallResult")
+        return tabbing + "# Modify AdvertisingIdType:  Prevents us from sending the id multiple times, and allows automated tests to determine id was sent successfully\n"
+            + tabbing + "AdvertisingIdType += \"_Successful\"\n";
+    else if (apiCall.result === "GetEntityTokenResponse")
+        return tabbing + "PlayFabSettings.EntityToken = playFabResult[\"EntityToken\"] or PlayFabSettings.EntityToken\n";
+    return "";
+}
+
+function getDeprecationAttribute(tabbing, apiObj) {
+    var isDeprecated = apiObj.hasOwnProperty("deprecation");
+    var deprecationTime = null;
+    if (isDeprecated)
+        deprecationTime = new Date(apiObj.deprecation.DeprecatedAfter);
+    var isError = isDeprecated && (new Date() > deprecationTime) ? "true" : "false";
+
+    if (isDeprecated && apiObj.deprecation.ReplacedBy != null)
+        return tabbing + "# [Obsolete(\"Use '" + apiObj.deprecation.ReplacedBy + "' instead\", " + isError + ")]\n";
+    else if (isDeprecated)
+        return tabbing + "# [Obsolete(\"No longer available\", " + isError + ")]\n";
+    return "";
+}
+
+function generateApiSummary(tabbing, apiElement, summaryParam, extraLines) {
+    var lines = generateApiSummaryLines(apiElement, summaryParam, extraLines);
+
+    var output;
+    if (lines.length === 1) {
+        output = tabbing + "# " + lines.join("\n" + tabbing + "# ") + "\n";
+    } else if (lines.length > 0) {
+        output = tabbing + "# " + lines.join("\n" + tabbing + "# ") + "\n";
+    } else {
+        output = "";
+    }
+    return output;
+}
+
+function getComparator(tabbing, dataTypeName, dataTypeSortKey)
+{
+    //var output = multiTab(tabbing, 3) + "def __eq__(self, " + dataTypeName + " other):\n" +
+    var output = multiTab(tabbing, 2) + "def __eq__(self, other):\n" +
+        multiTab(tabbing, 3) + "if other == None or other." + dataTypeSortKey + " == None:\n" +
+        multiTab(tabbing, 4) + "return 1\n" +
+        multiTab(tabbing, 3) + "if " + dataTypeSortKey + " == None:\n" +
+        multiTab(tabbing, 4) + "return -1\n"+
+        multiTab(tabbing, 3) + "return "+dataTypeSortKey+".__eq__(self."+dataTypeSortKey+", other."+dataTypeSortKey+")\n";
+
+    return output;
+}
+
+function multiTab(tabbing, numTabs)
+{
+    var finalTabbing = "";
+    while (numTabs != 0)
+    {
+        finalTabbing += tabbing;
```

Comment:
This is one of the highest performance cost ways to build a string.
I should tell you about how awful strings are sometime.
Remind me outside the context of this review.
We'll leave it for now, but we'll probably fix it separately later.